source/linux/basic/installing_uninstalling_software.rst

```diff
@@ -0,0 +1,74 @@
+Installing and Uninstalling Software
+====================================
+
+Most Linux distributions come with a package manager like apt or yum. It's made to simplify the process of downloading and installing new software and its dependencies on a users system. 
+In the following only debian based systems are covered because the VM you get from PHYTEC by purchasing a Kit is a debian based Distribution.
```

Comment:
Maybe more like: 
....
The following sections only cover Debian based systems, since the PHYTEC Development Environment, which ships with all PHYTEC kits, uses a virtual machine (VM) with a Debian based Linux distribution.
...